### PR TITLE
INGM-303 Add change_node_id function

### PR DESCRIPTION
**Pull request by MartinALL91**
### Description

Fixes #INGM-303

### Type of change

- Add change_node_id function

### CODE REVIEW

**Review by WoodElv — APPROVED** [review]

File: ingeniamotion/configuration.py [hunk 1]
```diff
@@ -979,3 +979,27 @@ def change_baudrate(self, baud_rate: CAN_BAUDRATE, servo: str = DEFAULT_SERVO) -
...
+        net = self.mc._get_network(servo)
+        if not isinstance(net, CanopenNetwork):
+            raise ValueError(f"Servo {servo} is not a CANopen device.")
+        vendor_id = self.get_vendor_id(servo)
+        (
+            (prod_code, _),
+            (rev_number, _),
+            _,
+            (serial_number, _),
+        ) = self.get_drive_info_coco_moco(servo)
```
Comment (GullinBustin) [inline ingeniamotion/configuration.py:1004]: Use get_product_code, get_revision_number and get_serial_number functions instead get_drive_info_coco_moco to get the drive info. This change will reduce the drive calls

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: ingeniamotion/configuration.py (hunk 1 above)
Comment (MartinALL91) [reply ingeniamotion/configuration.py]: Done

**Review by GullinBustin — CHANGES_REQUESTED** [review]
Update CHANGELOG.md

**Comment by MartinALL91** [comment]
> Update CHANGELOG.md

Done

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**